qmetaobject/src/qtdeclarative.rs

```diff
@@ -502,16 +502,106 @@ pub fn qml_register_type<T: QObject + Default + Sized>(
     })
 }
 
-/// Wrapper around [`void qmlRegisterModule(const char *uri, int versionMajor, int versionMinor)`][qt] function.
+/// Register the given type as an uncreatable QML type
 ///
-/// [qt]: https://doc.qt.io/qt-5/qqmlengine.html#qmlRegisterModule
-#[cfg(qt_5_9)]
-pub fn qml_register_module(
+/// Refer to the Qt documentation for QQmlEngine::qmlRegisterUncreatableType.
+pub fn qml_register_uncreatable_type<T: QObject + Sized>(
     uri: &CStr,
     version_major: u32,
     version_minor: u32,
+    qml_name: &CStr,
+    no_creation_reason: &CStr,
```

Comment:
This should be a QString, since it is a user visible string, and that's also what Qt uses.

Reply:
Ah, you're right. I was following the pattern of the other CStr arguments, but Qt does seem to use both raw C strings and QStrings for whatever reason.

Fixed.

Reply:
> since it is a user visible string

I don't think that argument is really any valid, to be honest :)
Can't imagine a user reading the "uncreatable type" error at runtime.